packages/core/src/components/DismissableBanner/DismissableBanner.tsx

```diff
@@ -60,17 +63,39 @@ type Props = {
 
 const DismissableBanner: FC<Props> = ({ variant, /* setting, */ message }) => {
   // const [show, setShown, loading] = useSetting(setting);
-  const [show, setShown] = useState(true);
   const classes = useStyles();
+  const storageApi = useApi(storageApiRef);
+  const settingsStore = storageApi.forBucket('settings');
```

Comment:
You can also create a new namespace here rather than settings. So maybe `forBucket(‘notifications’)` is more semantic